qcodes/instrument_drivers/tektronix/Keithley_6500.py

```diff
@@ -167,11 +167,12 @@ def __init__(self, name, address, reset_device=False, **kwargs):
                                             'bus': 'COMM',
                                             'external': 'EXT'})
 
+        # Control interval between scans, default is 0.
```

Comment:
I meant to add this part :) :
```suggestion
        # Control interval between scans; the default value from the instrument is 0,
        # hence 0 is included in the validator's range of this parameter.
```